templates/e-commerce/partials/blog-preview.html

```diff
@@ -0,0 +1,11 @@
+{{#articles}}
+<article class="md-pb7">
+  <amp-img class="mb3 md-col-10" src="{{images.0.narrow}}" srcset="{{images.0.narrow}} 750w,{{images.0.wide}} 1280w" width="{{imageWidth}}" height="{{imageHeight}}" layout="responsive" alt="{{title}}"></amp-img>
```

Comment:
This is to target only the first element in the array. I'm using the same 'blog' object for both the blog listings and blog article. Only the first blog has the full article data in it. Therefore by using `.0` we are able to target it.